src/main/java/com/arpnetworking/metrics/mad/parsers/JsonToRecordParser.java

```diff
@@ -104,6 +106,8 @@ public Record parse(final byte[] data) throws ParsingException {
         final String version = versionNode.get().textValue().toLowerCase(Locale.getDefault());
         try {
             switch (version) {
+                case "2g":
+                    return parseV2gLogLine(jsonNode);
```

Comment:
@BrandonArp do we need a Steno2g format too? I can't remember the context around Steno...